docker-compose.yml

```diff
@@ -2,19 +2,10 @@ version: '2'
 services:
     jenkins:
         build: .
-        depends_on: 
-            - remoting-kafka-agent
         ports:
             - "8080:8080"
-            - "50000:50000"
+            - "50000:50000"       
             - "36088:36088"
-    
-    remoting-kafka-agent:
```

Comment:
Why have you removed it?

Reply:
Maybe it makes sense to create a task for Official Remoting Kafka Agent packaging and configure build on DockerHub